src/ChatGptNet/ChatGptClient.cs

```diff
@@ -197,6 +203,41 @@ public Task DeleteConversationAsync(Guid conversationId, bool preserveSetup = fa
         return Task.CompletedTask;
     }
 
+    public Task<Guid> LoadConversationAsync(IEnumerable<ChatGptMessage> messages)
+    {
+        return LoadConversationAsync(new Guid(), messages);
+    }
```

Comment:
This method should be put in the `IChatGtpClient` interface, like this one: https://github.com/marcominerva/ChatGptNet/blob/5539626076a4d1df9f6acb5765b1046b5261eacf/src/ChatGptNet/IChatGptClient.cs#L19-L20